src/SignalR.Orleans/Extensions.cs

```diff
@@ -1,21 +1,20 @@
-using System;
-using System.Reflection;
 using Microsoft.AspNetCore.SignalR;
 using Orleans;
 using Orleans.Hosting;
 using Orleans.Runtime.Configuration;
 using Orleans.Serialization;
 using SignalR.Orleans;
 using SignalR.Orleans.Clients;
-using SignalR.Orleans.Groups;
+using System.Reflection;
 
 namespace Microsoft.Extensions.DependencyInjection
 {
     public static class OrleansDependencyInjectionExtensions
     {
         public static ClusterConfiguration AddSignalR(this ClusterConfiguration config)
         {
-            config.Globals.SerializationProviders.Add(typeof(HubMessageSerializer).GetTypeInfo());
+            // TODO: Check with @galvesribeiro if its needed, to be fixed or deleted.
+            // config.Globals.SerializationProviders.Add(typeof(HubMessageSerializer).GetTypeInfo());
```

Comment:
@galvesribeiro, I've commented this since it was throwing an exception on `Silo` and `Client` start. Is this serializer really needed? if not let me know and I will delete it. 

Without this, it's working perfectly.

Reply:
I think with the `fallback serializer provider` workaround, we don't need a custom serializer. I will continue doing testing without it and I will let you know how it goes.

The exception is:
```
|fail: Orleans.Serialization.SerializationManager[102404]
      Failed to create instance of type: SignalR.Orleans.HubMessageSerializer
System.MissingMethodException: No parameterless constructor defined for this object.
   at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean& canBeCached, RuntimeMethodHandleInternal& ctor)
   at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, StackCrawlMark& stackMark)
   at System.Activator.CreateInstance(Type type, Boolean nonPublic)
   at System.Activator.CreateInstance(Type type)
   at Orleans.Serialization.SerializationManager.<RegisterSerializationProviders>b__99_0(TypeInfo typeInfo)
info: Orleans.OutsideRuntimeClient[100313]
```